src/main/resources/log4j2.yml

```diff
@@ -9,8 +9,8 @@ Configuration:
         # TODO: determine if the following should be added:
         #   1. request timer (e.g. %9cmdc{requestTimer}); should reference RequestDurationPatternConverter but not working
         #   2. IP address (e.g. %15X{ipAddress}); will this show us real remote IP in docker containers?
-        #   3. session ID (e.g. sid:%-5X{sessionId}); is this useful since many containerized services will be stateless?
-        pattern: "%highlight{%d{yyyy-MM-dd HH:mm:ss.SSS} [rid:%5X{requestId}] %-5level %c{1}:%L - %m%n}"
+        #   3. Trace ID (e.g. traceId:%-5X{sessionId}); A distributed trace identifier that tracks a request throughout services
```

Comment:
I don't think you need to comment here since we are already incorporating it.  Note 1 and 2 are TBD.